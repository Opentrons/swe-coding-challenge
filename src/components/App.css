:global(*) {
  box-sizing: border-box;
  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
}

.app {
  padding: 1rem;
}

.app_title {
  margin-top: 0.5rem;
  margin-bottom: 1rem;
}

.app_contents {
  height: 16rem;
}
