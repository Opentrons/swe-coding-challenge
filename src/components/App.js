import React from 'react'
import { hot } from 'react-hot-loader/root'

import styles from './App.css'

export function App() {
  return <main className={styles.app}>hello world</main>
}

export default hot(App)
