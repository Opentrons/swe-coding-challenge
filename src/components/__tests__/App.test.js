import React from 'react'
import { shallow } from 'enzyme'

import App from '../App'

it('component renders', () => {
  const tree = shallow(<App />)
  expect(tree).toMatchSnapshot()
})
