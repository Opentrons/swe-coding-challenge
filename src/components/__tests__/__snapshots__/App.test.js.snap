// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component renders 1`] = `
<main
  className="app"
>
  hello world
</main>
`;
